src/prefect/agent/local/agent.py

```diff
@@ -105,6 +105,7 @@ def __init__(
                 "github-flow-storage",
                 "webhook-flow-storage",
                 "gitlab-flow-storage",
+                "bitbucket-flow-storage",
```

Comment:
We're planning on removing these default labels in a near future (0.14.0) release so this can be left out since it will be gone soon anyway and we can avoid making it another breaking change 🙂